sinks/types.go

```diff
@@ -15,8 +15,15 @@
 // This package implements the various storage backends supported by heapster.
 package sinks
 
+import "github.com/GoogleCloudPlatform/heapster/sinks/api/v1"
+
 // TODO: Once in-memory storage is implemented, make the manager extract data from the in-memory store periodically.
-type ExternalSinkManager interface {
+type Sink interface {
 	Store(input interface{}) error
 	DebugInfo() string
 }
+
+type ExternalSinkManager interface {
+	Sink
+	SetSinks([]v1.ExternalSink) error
```

Comment:
We should document this (don't follow the bad example we set above :) ). Lets also documen tthe ExternalSinkManager